build.sbt

```diff
@@ -145,29 +134,19 @@ lazy val rocketchip = freshProject("rocketchip", rocketChipDir)
   )
 lazy val rocketLibDeps = (rocketchip / Keys.libraryDependencies)
 
-// -- Chipyard-managed External Projects --
-
-lazy val firrtl_interpreter = (project in file("tools/firrtl-interpreter"))
-  .sourceDependency(firrtlRef, firrtlLib)
-  .settings(commonSettings)
-  .settings(libraryDependencies ++= (Global / firrtlLibDeps).value)
-lazy val firrtlInterpreterLibDeps = (firrtl_interpreter / Keys.libraryDependencies)
 
-lazy val treadle = (project in file("tools/treadle"))
-  .sourceDependency(firrtlRef, firrtlLib)
-  .settings(commonSettings)
-  .settings(libraryDependencies ++= (Global / firrtlLibDeps).value)
-lazy val treadleLibDeps = (treadle / Keys.libraryDependencies)
+// -- Chipyard-managed External Projects --
 
+// Because we're not using a release version of iotesters to work around a
+// scala test version problem, override it's libdeps to prevent using snapshots
 lazy val chisel_testers = (project in file("tools/chisel-testers"))
```

Comment:
Why can't we make this a published dep? Is this comment a recent problem with the 3.5 bump? I guess can we bump scala test and use the pub. version?

Reply:
In the latest release, Chisel testers still brings in a incompatible version of scalatest. The submodule still points at a branch that manually changes this. We can fix this in the 3.5 bump.

Reply:
It is possible to override transitive dependencies, but maybe not worth it since this should be further fixed in 3.5 bump.